apps/pie-docs/src/components/button/code/props.json

```diff
@@ -41,133 +54,188 @@
             "type",
             {
                 "type": "code",
-                "item": ["buttom", "reset", "submit"]
+                "item": [
+                    "buttom",
+                    "reset",
+                    "submit"
+                ]
             },
             "Set the type of the button.",
             {
                 "type": "code",
-                "item": ["submit"]
+                "item": [
+                    "submit"
+                ]
             }
         ],
         [
             "variant",
             {
                 "type": "code",
-                "item": ["destructive", "destructive-ghost", "ghost-inverse", "inverse", "ghost", "outline-inverse", "outline", "secondary", "primary"]
+                "item": [
+                    "primary",
+                    "secondary",
+                    "outline",
+                    "outline-inverse",
+                    "ghost",
+                    "inverse",
+                    "ghost-inverse",
+                    "destructive",
+                    "destructive-ghost"
+                ]
             },
             "Set the variant of the button.",
             {
                 "type": "code",
-                "item": ["primary"]
+                "item": [
+                    "primary"
+                ]
             }
         ],
         [
             "disabled",
             {
                 "type": "code",
-                "item": ["true", "false"]
+                "item": [
+                    "true",
+                    "false"
+                ]
             },
             "if <b>true</b>, disables the button.",
             {
                 "type": "code",
-                "item": ["false"]
+                "item": [
+                    "false"
+                ]
             }
         ],
         [
             "isFullWidth",
             {
                 "type": "code",
-                "item": ["false", "true"]
+                "item": [
+                    "false",
```

Comment:
Can we agree to be consistent for booleans and have `true` always first?

Reply:
I don't know why, but this [just reminded me of shooting stars](https://www.youtube.com/watch?v=57PTxe2hEqc) 😆